src/styles/card.scss

```diff
@@ -0,0 +1,117 @@
+.card {
+  width: 200px;
+  height: 408px;
+  background-color: $white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: block;
+  position: relative;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+}
+
+.card__img {
+  height: 134px;
+  width: 160px;
+  display: block;
+  position: relative;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 40px;
+}
+
+.card__name {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: $main-accent;
+  margin-top: 0;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: $secondary;
+  margin-bottom: 16px;
+}
+
+.card__review {
+  margin-bottom: 24px;
+  display: flex;
+  justify-content: space-between;
+}
+
+.card__review-stars {
+  display: flex;
+}
+
+.stars-star {
+  background-repeat: no-repeat;
+  display: flex;
+  background-image: url(../images/star.svg);
+  width: 16px;
+  height: 16px;
+  background-position: center;
+}
+
+.stars-4 .stars-star:nth-child(-n + 4) {
```

Comment:
![image](https://github.com/mate-academy/layout_product-cards/assets/102289654/6f668fc0-80c9-4672-8e2f-97de85690c10)
this will not work as you gave the styles only for 4 stars. you should reuse the whole stars styles